common.go

```diff
@@ -1,6 +1,7 @@
-// common.go defines common types used throughout the OpenAI API.
 package openai
 
+// common.go defines common types used throughout the OpenAI API.
```

Comment:
why this change?

Reply:
Ah, if that comment is above the package name, it shows up in the go docs package description. It didnt look like it was meant to be there